### DISCUSSION TITLE
can't login oauth google because there is #/ in the url

### DISCUSSION THREAD

**Discussion by bobwatcherx** [quasarframework/quasar#15149] [General - Components / Directives / etc]
I made a Google oauth login, and there doesn't seem to be any error in the browser.  and after i click login button .  but a white screen occurs, and there is no process like logging in to Google.  is there something wrong , or is there a workaround for Quasar js ?

my boot file :
```
import { boot } from 'quasar/wrappers'
import GoogleSignInPlugin from "vue3-google-signin"

export default boot(({app})=>{

const gauthClientId = "1029569400250-nv24bm9gf29n3guamqvg2366qr9mb94k.apps.googleusercontent.com"
  app.use(GoogleSignInPlugin, {
  clientId:gauthClientId,
});
})
```
my Indexpage.vue
```
<script setup lang="ts">
import {
  GoogleSignInButton,
  type CredentialResponse,
} from "vue3-google-signin";

// handle success event
const handleLoginSuccess = (response: CredentialResponse) => {
  const { credential } = response;
  console.log("Access Token", credential);
};

// handle an error event
const handleLoginError = () => {
  console.error("Login failed");
};
</script>

<template>
  <GoogleSignInButton
    @success="handleLoginSuccess"
    @error="handleLoginError"
  ></GoogleSignInButton>
</template>
```
**Authorized JavaScript origins : http://localhost:9000
Authorized redirect URIs : http://localhost:9000/**

**I don't think Google recognizes the #/ sign because my settings in the Google Cloud console for the authorized callback url don't use #**

whereas the #/ sign cannot be used on Authorized redirect URIs

**Comment by metalsadman** [comment]
use vue-router's history mode. `vueRouterMode?: "hash" | "history";` https://quasar.dev/quasar-cli-vite/quasar-config-js#build
https://router.vuejs.org/guide/essentials/history-mode.html#example-server-configurations
